src/features/chat/chat.ts

```
@@ -411,107 +423,148 @@ export class Chat {
+        },
+        async (payload: { new: { type: any; data: any; }; }) => {
+          const { type, data } = payload.new;
+          console.log("Realtime message received:", type, data);
+
+          switch (type) {
+            case "normal":
+              const messages: Message[] = [
+                { role: "system", content: config("system_prompt") },
+                ...this.messageList!,
+                { role: "user", content: data },
+              ];
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)